kamailio/local.cfg

```diff
@@ -27,8 +27,8 @@ debug = L_INFO
 # # #!trydef PUSHER_ROLE
 # # #!trydef REGISTRAR_SYNC_ROLE
 # # #!trydef PRESENCE_SYNC_ROLE
-# # #!trydef PRESENCE_NOTIFY_SYNC_ROLE
-# # #!trydef PRESENCE_BROADCAST_ROLE
+#!trydef PRESENCE_NOTIFY_SYNC_ROLE
+#!trydef PRESENCE_NAT_ROLE
```

Comment:
@lazedo we should discuss this, I would prefer to avoid changing local.cfg as it makes updating more difficult.  If we do proceed we should move them to the enabled roles section above at a minimum ;)